runtime/onert/api/include/nnfw_experimental.h

```diff
@@ -220,6 +220,15 @@ typedef struct nnfw_train_info
   NNFW_TRAIN_OPTIMIZER opt = NNFW_TRAIN_OPTIMIZER_SGD;
 } nnfw_train_info;
 
+/**
+ * @brief Set train_info into session
+ * @note  This function should be called after calling {@link nnfw_load_model_from_file}
+ *        and before calling {@link nnfw_prepare}
```

Comment:
Maybe?
```suggestion
 *        and before calling {@link nnfw_train_prepare}
```

BTW, is there anything else to be modified in `nnfw_train_prepare` due to this commit?

Reply:
Ah, Updated. 

> BTW, is there anything else to be modified in nnfw_train_prepare due to this commit? 

I'd like to update `nnfw_train_prepare` like this.
* remove `nnfw_train_info` argument and update to use session's train_info.  
https://github.com/Samsung/ONE/pull/12383/files#r1437471834

Reply:
I see, there are changes in the draft PR. I will review it later on the PR that it will be applied.